Makefile

```diff
@@ -83,12 +84,12 @@ wavs-cli:
 
 ## upload-component: uploading the WAVS component | COMPONENT_FILENAME, WAVS_ENDPOINT
 upload-component:
-# TODO: move to $(WAVS_CMD)  upload-component ./compiled/${COMPONENT_FILENAME}
+# TODO: move to `$(WAVS_CMD) upload-component ./compiled/${COMPONENT_FILENAME} --wavs-endpoint ${WAVS_ENDPOINT}`
 	@wget --post-file=./compiled/${COMPONENT_FILENAME} --header="Content-Type: application/wasm" -O - ${WAVS_ENDPOINT}/upload | jq -r .digest
 
-## deploy-service: deploying the WAVS component service json | SERVICE_CONFIG_FILE, CREDENTIAL, WAVS_ENDPOINT
+## deploy-service: deploying the WAVS component service json | SERVICE_URL, CREDENTIAL, WAVS_ENDPOINT
 deploy-service:
-	@$(WAVS_CMD) deploy-service-raw --service `jq . -cr ${SERVICE_CONFIG_FILE}` $(if $(WAVS_ENDPOINT),--wavs-endpoint $(WAVS_ENDPOINT),) --log-level=info --data /data/.docker --home /data $(if $(CREDENTIAL),--eth-credential $(CREDENTIAL),)
+	@$(WAVS_CMD) deploy-service --service-url "$(SERVICE_URL)" --log-level=debug --data /data/.docker --home /var/wavs $(if $(WAVS_ENDPOINT),--wavs-endpoint $(WAVS_ENDPOINT),) $(if $(CREDENTIAL),--evm-credential $(CREDENTIAL),)
```

Comment:
this requires /var/wavs home now instead of /data. did the env now work or is this actually correct now since we do not have cli.toml ?

Reply:
this looks right still, works